File: src/main/java/dev/slne/protect/bukkit/region/settings/ProtectionTransactionCause.java
package dev.slne.protect.bukkit.region.settings;

/**
 * Represents the protection transaction cause
 */
public enum ProtectionTransactionCause {
	/**
	 * The protection was created
	 */
	NEW_PROTECTION,

	/**
	 * The protection was expanded
	 */
	EXPAND_PROTECTION,

	/**
	 * The protection was renamed
	 */
	RENAMED_PROTECTION,

	/**
	 * The protection was sold
	 */
	SOLD_PROTECTION;
}

File: src/main/kotlin/dev/slne/surf/protect/paper/region/settings/ProtectionTransactionCause.kt
package dev.slne.surf.protect.paper.region.settings

/**
 * Represents the protection transaction cause
 */
enum class ProtectionTransactionCause {
    /**
     * The protection was created
     */
    NEW_PROTECTION,

    /**
     * The protection was expanded
     */
    EXPAND_PROTECTION,

    /**
     * The protection was renamed
     */
    RENAMED_PROTECTION,

    /**
     * The protection was sold
     */
    SOLD_PROTECTION
}
